dspace-api/src/test/java/org/dspace/statistics/export/ITExportUsageEventListener.java

```diff
@@ -0,0 +1,432 @@
+
+
+    protected CommunityService communityService = ContentServiceFactory.getInstance().getCommunityService();
+    protected ConfigurationService configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+    protected CollectionService collectionService = ContentServiceFactory.getInstance().getCollectionService();
+    protected ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+    protected InstallItemService installItemService = ContentServiceFactory.getInstance().getInstallItemService();
+    protected WorkspaceItemService workspaceItemService = ContentServiceFactory.getInstance().getWorkspaceItemService();
+    protected EPersonService ePersonService = EPersonServiceFactory.getInstance().getEPersonService();
+    protected GroupService groupService = EPersonServiceFactory.getInstance().getGroupService();
+    protected BundleService bundleService = ContentServiceFactory.getInstance().getBundleService();
+    protected BitstreamService bitstreamService = ContentServiceFactory.getInstance().getBitstreamService();
+    protected EntityTypeService entityTypeService = ContentServiceFactory.getInstance().getEntityTypeService();
+    protected FailedOpenURLTrackerService failedOpenURLTrackerService =
+            OpenURLTrackerLoggerServiceFactory.getInstance().getOpenUrlTrackerLoggerService();
+
+    protected ArrayList testProcessedUrls = DSpaceServicesFactory.getInstance().getServiceManager()
+                                                                 .getServiceByName("testProcessedUrls",
+                                                                                   ArrayList.class);
+
+    @Spy
+    ExportUsageEventListener exportUsageEventListener;
+
+    private Item item;
+    private Item itemNotToBeProcessed;
+    private Bitstream bitstream;
+    private Bitstream bitstreamNotToBeProcessed;
+    private EntityType entityType;
+    private Community community;
+    private Collection collection;
+
+    private String encodedUrl;
+    private String encodedUIUrl;
+
+
+    /**
+     * Initializes the test by setting up all objects needed to create a test item
+     */
+    @Before()
+    public void init() {
+        super.init();
+
+        configurationService.setProperty("stats.tracker.enabled", true);
+        configurationService.setProperty("stats.tracker.type-field", "dc.type");
+        configurationService.setProperty("stats.tracker.type-value", "Excluded type");
+
+
+        context.turnOffAuthorisationSystem();
+        try {
+            exportUsageEventListener.configurationService = configurationService;
+
+            entityType = entityTypeService.create(context, "Publication");
+            community = communityService.create(null, context);
+            collection = collectionService.create(context, community);
+            item = installItemService.installItem(context, workspaceItemService.create(context, collection, false));
```

Comment:
Please update `init()` to use Builder classes (which have now moved to `dspace-api`...see #2897.  That will allow you to simplify the `destroy()` method.